CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ v2.1.0 [2019-11-21]
 ### Bugfixes
 - [#375](https://github.com/latera/homs/pull/375) Adjust new elements with existent style.
 
+v2.0.1 [unreleased]
+-------------------
+### Bugfixes
```

Comment:
We write about bugfixes one time, no?

Reply:
Okay. I just thought it would be strange if we released v2.0.1 without changes in changelog.